CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ target_link_libraries(project1 project1_lib)
 ################################
 if (test)
   # This adds another subdirectory, which has 'project(gtest)'.
+  set( gtest_force_shared_crt ON CACHE BOOL "Always use msvcrt.dll" )
```

Comment:
Hey, thanks for the PR. Can you add a comment explaining this line? eg., say it's only for Visual C++